playbooks/roles/scorm/tasks/main.yml

```diff
@@ -1,101 +1,22 @@
 # CONFIGURE NGINX
 # SYNC LOCAL SCORM CONTENT PACKAGES WITH S3 IF NOT STORED THERE TO BEGIN WITH
 
-- name: FIXME create application user (copied over from edxapp/tasks/main.yml)
-  user: >
-    name="{{ edxapp_user }}" home="{{ edxapp_app_dir }}"
-    createhome=no shell=/bin/false
-  tags:
-    - install
-    - install:base
+# do these as includes since Vagrant  < 2.0 doesn't obey
+# conditionals to include the whole scorm role
 
-- name: create all service directories
-  file: >
-    path="{{ item.value.path }}"
-    state="directory"
-    owner="{{ item.value.owner }}"
-    group="{{ item.value.group }}"
-    mode="{{ item.value.mode }}"
-  with_dict: scorm_dirs
-  when: EDXAPP_XBLOCK_SETTINGS.ScormXBlock
+- name: define scorm config from settings if available
+  set_fact: 
+    scorm_config: "{{ EDXAPP_XBLOCK_SETTINGS.ScormXBlock|default('{}') }}"
```

Comment:
Does this expression work? (I don't know). If it does then keep it.

Reply:
Yes, it works :) .  Just ran it yesterday.